### DISCUSSION TITLE
How to add an hint for `A<CR>` ?

### DISCUSSION THREAD

**Discussion by mb-915** [m4xshen/hardtime.nvim#112] [Q&A]
Hello, 

I want to add an hint to use `o` instead of `A<CR>` but I have two problems . Here it what I've tried:

```lua
    hints = {
      ["A<CR>"] = {
        message = function() return "Use o instead" end,
        length = 2,
      },
    }
 ```

even when trying to map each key individually I could not get them to work : I did not manage to create an hint using `["<CR>"]`, and I did not manage to map `["Ai"]` (`A` alone get me the hint message, but `A` + another key typed in insert mode does not work).

Does anyone know how to add an hint for this combo?

**Comment by henriqueleng** [comment]
I tried to do the same before reading here. I could not get it to detect a newline in insert mode, only when in normal mode.

**Comment by m4xshen** [comment]
Hardtime currently doesn't support hints in INSERT mode so that config won't work. I'll consider adding INSERT mode hints support in the future.
